analyzer/db_manager.py

```diff
@@ -0,0 +1,24 @@
+from analyzer.postgres_db import get_db
+
+
+def connection_handler(func):
+    def wrapper(*args, **kwargs):
+        with get_db() as conn:
+            with conn.cursor() as cur:
+                res = func(cur, *args, **kwargs)
+        return res
+    return wrapper
+
+
+@connection_handler
+def get_records_by_numbers(cur, numbers):
+    param_dict = {str(k): v for k, v in enumerate(numbers)}
```

Comment:
enumerate() called three times an the same iterable
```suggestion
    enum = enumerate(numbers)
    param_dict = {str(k): v for k, v in enum}
```